.teamcity/settings.kts

```diff
@@ -859,9 +859,8 @@ object RunCanaryE2eTests : BuildType({
 
 				export LIVEBRANCHES=true
 				export NODE_CONFIG_ENV=test
-
-				## Uncomment to debug Magellan
-				#export MAGELLANDEBUG=true
+				export MAGELLANDEBUG=true
```

Comment:
Did you mean to leave this on?

Reply:
Yes! That's what triggers chrome and chromedriver logs